pl_bolts/models/self_supervised/swav/swav_module.py

```diff
@@ -183,8 +183,8 @@ def on_train_epoch_start(self):
                     self.feat_dim,
                 )
 
-                if self.gpus > 0:
-                    self.queue = self.queue.cuda()
+            if self.gpus > 0 and self.queue is not None:
+                self.queue = self.queue.cuda()
```

Comment:
```suggestion
            self.queue is not None:
                self.queue = self.queue.to(self.device)
```
this is device agnostic, no need to check the gpus argument.